packages/framework/src/Model/Mail/MailTemplateDataFactory.php

```diff
@@ -88,4 +93,33 @@ public function createFromOrderStatuses(array $orderStatuses, array $mailTemplat
 
         return $orderStatusMailTemplatesData;
     }
+
+    /**
+     * @param string|null $body
+     * @param int $domainId
+     * @return string
+     */
+    protected function ensureBodyIsInCorrectGrapeJsFormat(
+        ?string $body,
+        int $domainId = Domain::FIRST_DOMAIN_ID,
+    ): string {
+        $locale = $this->domain->getDomainConfigById($domainId)->getLocale();
+
+        if ($body === null || ctype_space($body) || strip_tags($body) === '') {
```

Comment:
Certain content (like `<div>    </div>`, for example) can still cause trouble.

what about 
```suggestion
        if ($body === null || trim(strip_tags($body)) === '') {
```

Reply:
I have reworked it a bit more. I have removed box-content and added sizing for editable div in Grapes, so it is always visible. I have made those changes directly in history, as there is not much code and I find it easier to review again.